app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/itemlisting/ItemListingScreen.kt

```diff
@@ -197,20 +211,24 @@ fun ItemListingScreen(
                         )
                     }
                 },
+                onDownloadBitwardenClick = remember(viewModel) {
+                    {
+                        viewModel.trySendAction(ItemListingAction.DownloadBitwardenClick)
+                    }
+                },
+                onDismissDownloadBitwardenClick = remember(viewModel) {
+                    {
+                        viewModel.trySendAction(ItemListingAction.DownloadBitwardenDismiss)
+                    }
+                },
             )
         }
 
-        is ItemListingState.ViewState.Error -> {
-            Text(
-                text = "Error! ${currentState.message}",
-                modifier = Modifier.fillMaxSize(),
-            )
-        }
-
```

Comment:
I got rid of the Error and no network states, since they were technically impossible to show. 

If you think that's going too far and it'd be nice to have them around in case we want them later, I could add them back. But for now I wanted to only worry about the possible states 👍